### ISSUE TITLE
--no-online check returns an ASSERT if DB is running

### ISSUE THREAD

**Issue by jasonodonnell** [pgbackrest/pgbackrest#564]
Please provide the following information when submitting an issue (feature requests or general comments can skip this):

1. pgBackRest version:
2.0.4

2. PostgreSQL version:
10.4

3. Operating system/version - if you have more than one server (for example, a database server, a repository host server, one or more standbys), please specify each:
CentOS7 and RHEL7

4. Did you install pgBackRest from source or from a package?
RPM - PGDG and Crunchy

5. Please attach the following as applicable:
```bash
bash-4.2$ pgbackrest --stanza=db --no-online check
ASSERT [025]: : option archive-check is required
bash-4.2$ pgbackrest --stanza=db --no-online --no-archive-check check
ERROR: [031]: option 'archive-check' not valid without option 'online'
```

6. Describe the issue:
I was testing if `check` could reliably tell me if a stanza has been created or not via return codes.  During my testing I tried to run with `--no-online` flags and pgbackrest returned an ASSERT.  I will use the `info` command to do this test, however, I wanted to report this ASSERT coming back to the user.

**Comment by dwsteele** [comment]
It throws a regular error if the database is *not* running:
```
ERROR: [056]: Database: 9.4 6585217104678667162 could not connect to server: No such file or directory
            	Is the server running locally and accepting
            	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
```
I don't think this command should be accepting --no-online as an option.  May be an oversight in the config definition.

**Comment by dwsteele** [comment]
Fixed at 343424009795014bce2474f4 by removing the `online` option from the `check` command.
